src/triggered/wizards.ts

```diff
@@ -0,0 +1,195 @@
+import { html } from 'lit-element';
+import { get } from 'lit-translate';
+
+import '../filtered-list.js';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7203429)
```suggestion
import "../filtered-list.js";
```